hera_pspec/pspecdata.py

```diff
@@ -1399,8 +1401,8 @@ def pspec(self, bls1, bls2, dsets, pols, input_data_weight='identity', norm='I',
                     nsamp1 = np.sum(dset1.get_nsamples(bl1 + (p[0],))[:, self.spw_range[0]:self.spw_range[1]] * wgts1, axis=1) / np.sum(wgts1, axis=1).clip(1, np.inf)
                     nsamp2 = np.sum(dset2.get_nsamples(bl2 + (p[1],))[:, self.spw_range[0]:self.spw_range[1]] * wgts2, axis=1) / np.sum(wgts2, axis=1).clip(1, np.inf)
 
-                    # take average of nsamp1 and nsamp2 and multiply by integration time [seconds] to get total integration
-                    pol_ints.extend(np.mean([nsamp1, nsamp2], axis=0) * dset1.integration_time)
+                    # take inverse average of nsamp1 and nsamp2 and multiply by integration time [seconds] to get total integration
```

Comment:
Why are we taking the inverse average now? It would be good to add a comment to document why we need to do this.

Reply:
sure. the inverse average is done b/c the `nsample` is actually a proxy for `1/sigma`, where `sigma` is the std of the noise. so we really should be averaging `sigma` directly, which is equivalent to inversely averaging `nsamples`. I'll make a comment explaining this.